DAQ_System/Source/main.cpp

```diff
@@ -1,10 +1,82 @@
 #include "mbed.h"
+#include "LSM303DLHC.h"
+#include <iostream>
+
+static float _lsm303Accel_MG_LSB     = 0.001F;
+const int sampleCount = 100;
+//this should be based on a calibrated value for 1.0 G
+const double gravity = 1.0;
+
+//the following can be used for calibration
+int loopNum = 0;
+double average = 0;
+double previousSamples[sampleCount];
```

Comment:
Use snake case for variable names. Example: `loop_num`

On another note: `average` what? Be descriptive